spec.md

```diff
@@ -680,21 +680,16 @@ corresponding Service Instance - it is an implementation choice. Doing so would
 cause Platforms to serialize multiple binding creation requests when they are
 directed at the same Service Instance if concurrent updates are not supported.
 
-## Polling Last Operation
+## Polling Last Operation for Service Instances
 
 When a Service Broker returns status code `202 Accepted` for
 [Provision](#provisioning), [Update](#updating-a-service-instance), or
 [Deprovision](#deprovisioning), the Platform will begin polling the
 `/v2/service_instances/:instance_id/last_operation` endpoint to obtain the
-state of the last requested operation. The Service Broker response MUST contain
```

Comment:
For those following later on..... we removed this sentence to remove redundant info.